NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -676,7 +678,7 @@ class NeuroID
             showLogs = enable
         }
 
-        override fun getSDKVersion() = NIDVersion.getSDKVersion()
+        override fun getSDKVersion() = NIDVersion.getSDKVersion(isRN, isAdvancedDevice)
```

Comment:
This parameter is if they set isAdvanceDevice true, but that could be set to true and it is NOT the ADV library. I think we should utilize the reflection method to determine if the ADV library is even available to actually decide if they have ADV or not. Thoughts?

Reply:
I think it might be better to set the isAdvancedDevice to false if the capture fails then to make this correct.

Reply:
I am not opposed to doing that, if you want to update and make a note in the Android ticket and the iOS one so we can make sure they do the same thing